### DISCUSSION TITLE
Welcome to YTAD Discussions!

### DISCUSSION THREAD

**Discussion by rkrehn** [rkrehn/YTAD#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SignorCastello** [comment]
Thank you for this! I love seeing those little C# projects, they always work better than the most available ones. Kudos!

**Comment by DragonKing111222** [comment]
Hey there, I'm trying to download a playlist for Star Fox 64 Restored, and I have this issue every time.
I made sure I had ffmpeg, an empty folder selected, a cookies file that i grabbed directly from the playlist page, and followed all other instructions in the README as it was written. Despite this, every time I open the playlist/album in the downloader, all the songs show up (albeit with no artist or album name listed, but that doesn't bother me, I'm putting these into a rockbox mp3 player), but as soon as I click resume, all of them quickly disappear one by one and no songs get downloaded. The songs all say No under converted, but the program does seem to detect them in the playlist just fine (all the names of the songs are correct and so are the lengths of the songs). Any idea what's causing this?

**Reply by rkrehn** [reply]
Can you open the data folder and upload your error.log file? Also, mind sending me the playlist link?

**Reply by Galdinius** [reply]
I am facing the same issue as well

**Reply by Gastello** [reply]
> Can you open the data folder and upload your error.log file? Also, mind sending me the playlist link?



> Can you open the data folder and upload your error.log file? Also, mind sending me the playlist link?

2026-03-31 22:51:35 - System.InvalidOperationException: Uncommitted new row cannot be deleted.
   at System.Windows.Forms.DataGridViewRowCollection.RemoveAt(Int32 index)
   at YTPD.Form1.timer1_Tick(Object sender, EventArgs e) in C:\MEGA\visual basic\YTPD\Form1.cs:line 143

**Comment by hemofuru** [comment]
<img width="1005" height="556" alt="image" src="https://github.com/user-attachments/assets/3967df43-14aa-4d2f-bd35-9cb86fd76c6d" />
hi, this keeps happening eventho i already set the cookies as guided, the first song can be downloaded but the next one can't, can you help? i tried with others too and it seems it cant download more than 1 minutes

**Reply by hemofuru** [reply]
the duration is also wrong, like the 2nd song is supposed to be 4.02, and the 3rd one is supposed to be 4:56

**Comment by sweetestmarten** [comment]
I downloaded ffmpeg, it's defined in path, cookies should be working right. I hit resume and the first song is just sitting idly at 1%. I have attached the error log. 

[error.log](https://github.com/user-attachments/files/29228001/error.log)
<img width="1022" height="557" alt="Screenshot 2026-06-22 201434" src="https://github.com/user-attachments/assets/dc9d71ba-57dc-448f-9735-b3279b4d8e3d" />

**Reply by hemofuru** [reply]
up, still having the same problem, too
